packages/mds-logger/index.ts

```diff
@@ -19,27 +19,41 @@ import httpContext from 'express-http-context'
 const logger: Pick<Console, 'info' | 'warn' | 'error'> = console
 type LogLevel = keyof typeof logger
 
-const redact = (args: unknown[]): string[] =>
-  args.map(arg =>
-    JSON.stringify(arg instanceof Error ? arg.toString() : arg, (k, v) =>
-      ['lat', 'lng'].includes(k) ? '[REDACTED]' : v
-    )
+type LogArgs = [message: string, data?: Record<string, unknown> | Error]
+
+const redact = (arg: unknown): any => {
```

Comment:
Seems like the signature of `redact` could be tightened up now, but not a big deal.